packages/lit-dev-content/src/components/litdev-search.ts

```diff
@@ -240,8 +257,30 @@ export class LitDevSearch extends LitElement {
 
   private _listboxClicked = false;
 
-  update(changed: PropertyValues<this>) {
-    if (!this._searchText && this._inputEl) {
+  private _lastSuggestions: Suggestion[] = [];
+
+  private get _currentSuggestions() {
+    if (!this._searchTask || this._searchTask.status !== TaskStatus.COMPLETE) {
+      return this._lastSuggestions;
+    }
+
+    this._lastSuggestions = this._searchTask.value!;
+    return this._searchTask.value!;
+  }
+
+  private get _isExpanded() {
+    return (
+      this._isFocused &&
+      this._currentSuggestions.length > 0 &&
+      !this._closedByEscape
```

Comment:
I think another way to handle this, that seems to be consistent with some search interface like GitHub's, would be to just blur on escape. Then you can elide the `_closedByEscape` field.